include/lemlib/pose.hpp

```diff
@@ -25,43 +25,43 @@ class Pose {
          * @param other other pose
          * @return Pose
          */
-        Pose operator+(const Pose& other);
+        Pose operator+(const Pose& other) const;
         /**
          * @brief Subtract a pose from this pose
          *
          * @param other other pose
          * @return Pose
          */
-        Pose operator-(const Pose& other);
+        Pose operator-(const Pose& other) const;
         /**
          * @brief Multiply a pose by this pose
          *
          * @param other other pose
          * @return Pose
          */
-        float operator*(const Pose& other);
+        float operator*(const Pose& other) const;
         /**
          * @brief Multiply a pose by a float
          *
          * @param other float
          * @return Pose
          */
-        Pose operator*(const float& other);
+        Pose operator*(const float& other) const;
         /**
          * @brief Divide a pose by a float
          *
          * @param other float
          * @return Pose
          */
-        Pose operator/(const float& other);
+        Pose operator/(const float& other) const;
         /**
          * @brief Linearly interpolate between two poses
          *
          * @param other the other pose
          * @param t t value
          * @return Pose
          */
-        Pose lerp(Pose other, float t);
+        Pose lerp(Pose other, float t) const;
```

Comment:
this as well as `Pose::distance()` and `Pose::angle()` would benefit from taking `const Pose&`